src/Tasks/Microsoft.NET.Build.Tasks/GenerateRuntimeConfigurationFiles.cs

```diff
@@ -43,6 +43,8 @@ public class GenerateRuntimeConfigurationFiles : TaskBase
 
         public ITaskItem[] HostConfigurationOptions { get; set; }
 
+        public ITaskItem[] AdditionalProbingPaths { get; set; }
```

Comment:
I have a slight preference to keeping ITaskItem in case we ever want to add metadata to these items.